.github/workflows/ci.yml

```
@@ -22,6 +22,10 @@ jobs:
     - uses: actions/checkout@v4
       with:
         fetch-depth: 0
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)